src/pages/BrowserPage.qml

```diff
@@ -334,7 +337,11 @@ Page {
             bringToForeground(webView.chromeWindow)
             window.activate()
         }
-        onActivateNewTabViewRequested: activateNewTabView()
+        onActivateNewTabViewRequested: {
+            // Refuse if blocked by MDM
+            if (!AccessPolicy.browserEnabled) return
+            activateNewTabView()
+            }
```

Comment:
Wrong indentation.

Reply:
I moved this to the activateNewTabView() function anyway, to catch the cover action.